commons-numbers-complex/src/main/java/org/apache/commons/numbers/complex/Complex.java

```diff
@@ -2037,7 +1479,7 @@ public Complex atan() {
         // Define in terms of atanh
```

Comment:
This comment is not relevant any more. However it is missing from ComplexFunctions so you should move it there.